docs/sources/configure-client/aws-lambda.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "AWS Lambda Profiling Extension"
+menuTitle: "AWS Lambda Profiling Extension"
+description: "Profiling AWS Lambda functions with Pyroscope"
+weight: 100
+---
+
+# AWS Lambda Profiling Extension
```

Comment:
Because we use sentence case for headings, I've changed the capitalization of this heading.
```suggestion
# AWS Lambda profiling extension
```